<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送方</string>
  </property>
  <widget class="QPlainTextEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入待发送的数据&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_commit1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="frame_cat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>帧信息</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>85</width>
     <height>32</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sn = 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Sf">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sf = 0</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数据</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发送</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>接受</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
